plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -1297,13 +1299,14 @@ func TestAdmitLimitedResourceWithQuotaThatDoesNotCover(t *testing.T) {
 		},
 	}
 	kubeClient := fake.NewSimpleClientset(resourceQuota)
-	indexer := cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{"namespace": cache.MetaNamespaceIndexFunc})
 	stopCh := make(chan struct{})
 	defer close(stopCh)
 
 	informerFactory := informers.NewSharedInformerFactory(kubeClient, 0)
+	if err := informerFactory.Core().V1().ResourceQuotas().Informer().GetStore().Add(resourceQuota); err != nil {
+		t.Fatalf("Failed to add ResourceQuota(%q) to informer: %v", resourceQuota.ObjectMeta.Name, err)
+	}
```

Comment:
rather than directly manipulating the store, I think we should do the following after calling `createHandlerWithConfig`, in the same place we are calling `indexer.Add` today:

```
       informerFactory.Start(stopCh)
       informerFactory.WaitForCacheSync(stopCh)
```

Reply:
same comment applies to all places

Reply:
Started the informer in `createHandlerWithConfig` that let me remove all references to Indexer and Store. Now it is much easier to keep track of `ResourceQuota`s under test, because they need to be passed to `fake.Clientset` only